app/views/config_reports/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <%= if @host
   breadcrumbs do
-    [
+    { breadcrumbItems: [
       {caption: _("Hosts"), url: (url_for(hosts_path) if authorized_for(hash_for_hosts_path))},
       {caption: @host.name, url: (host_path(@host) if authorized_for(hash_for_host_path(@host)))},
       {caption: _('Reports'), url: url_for(config_reports_path(search: 'eventful=true'))},
-    ]
```

Comment:
this has no title in the breadcrums, it seems like api for reports index is not ideal as well, maybe we should disable it on this page for now.